trojsten/submit/views.py

```diff
@@ -238,6 +241,21 @@ def poll_submit_info(request, submit_id):
     }), content_type='application/json; charset=utf-8')
 
 
+def send_notification_email(submit):
+    send_mail(
+        _('[Trojstenweb] User submission detected'),
+        _('{name} submitted {possessive} solution to task {task}\n\nSubmit link: {submit_link}\n\nThis is an automated'
+          ' response, do not reply').format(
+            name=submit.user.get_full_name(),
+            possessive='his' if submit.user.gender == 'M' else 'her',
+            task=submit.task.__str__(),
+            submit_link='%s/admin/old_submit/submit/%s' % (settings.SITES[settings.SITE_ID].url, str(submit.id)),
```

Comment:
Na to, aby sme dostali URL k nejakému viewu sa v djangu používa funkica `reverse` https://docs.djangoproject.com/en/2.0/ref/urlresolvers/

URLky sú definované v jednom súbore urls a keď ich zmeníš tam, malo by všetko zostať fungovať - preto nechceš nikde inde v kóde používať urlky ako stringy.